plugins/module_utils/doc_fragments/instance_type_list.py

```diff
@@ -0,0 +1,37 @@
+"""Documentation fragments for the instance_type_list module"""
+
+specdoc_examples = ['''
+- name: List all of the Linode instance types
+  linode.cloud.instance_type_list: {}''', '''
+- name: Resolve all Linode instance types
+  linode.cloud.instance_type_list:
+    filter:
+      - name: class
+        values: nanode''']
+
+result_images_samples = ['''[
```

Comment:
Quick correction just for consistency:
`result_images_samples` -> `result_instance_types_samples`